OpenRA.Mods.Common/Traits/GiveCashOnCapture.cs

```diff
@@ -0,0 +1,56 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2016 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Mods.Common.Effects;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Lets the actor grant cash when captured.")]
+	class GiveCashOnCaptureInfo : UpgradableTraitInfo
```

Comment:
This will want to be `Gives` for consistency with `GivesBounty` and `GivesExperience`.